src/platform-includes/distributed-tracing/custom-instrumentation/dotnet.mdx

```diff
@@ -0,0 +1,75 @@
+On this page you will learn how to manually propagate trace information into and out of your .NET application. Please note, that you do not need to do this manually, if you [use one of our supported frameworks](/platforms/dotnet/usage/distributed-tracing/#how-to-use-distributed-tracing), or you have our <PlatformLink to="/performance/">performance monitoring feature</PlatformLink> turned on.
+
+To set it up manually, all you have to do is to make sure your application extracts incoming headers and to set those headers again when making an outgoing request within your application.
+
+## Step 1) Extract Incoming Tracing Information
+
+Incoming tracing information has to be extracted and stored in memory for later use. Sentry provides the `ContinueTrace()` function to help you with this. Tracing information can come from incoming headers, for example, by another Sentry SDK used in your frontend project.
+
+Here's an example using RabbitMQ of how to extract and store incoming tracing information using `ContinueTrace()`:
+
+```csharp
+using System.Text;
+using RabbitMQ.Client.Events;
+using Sentry;
+
+using var channel = ...;
+var consumer = new EventingBasicConsumer(channel);
+consumer.Received += (model, ea) =>
+{
+    // Extract headers from the incoming message
+    var headers = ea.BasicProperties.Headers;
+    if (headers != null)
```

Comment:
would u want a transaction even if there's no headers? But I guess for a demo itdoesn't matter